sdks/python/apache_beam/examples/streaming_wordcount_debugging.py

```diff
@@ -71,16 +72,14 @@ def process(
 class AddTimestampFn(beam.DoFn):
   """A DoFn that attaches timestamps to its elements.
 
-  It takes a string of integers and it attaches to each of them
-  a timestamp of its same value.
+  It just takes an element and adds a current timestamp.
 
-  For example, [120, 225, 312] will result in:
-  [(120, Timestamp(120)), (225, Timestamp(225)), (312, Timestamp(312))].
+  For example, Sometext will result in:
+  (Sometext, Timestamp(1234567890).
```

Comment:
Note that this change will make timestamp unpredictable would likely result in flaky tests.

Reply:
Fair point. The reason of that change is to make it work not only with int values, as an example is called wordcount. Will it be ok if we will use the positive hashed value of "Sometext" as a base for timestamp?

Reply:
That will be better by making tests reproducible. That will result in elements with timestamps across a wide range. Testers would not be able to predict what element will be in which window.

we change the input to be a tuple <message>,<timestamp> with a comma separation.

Reply:
Well, as this is an example of debugging functionality - more complicated logic could be added during real-time development. Just to keep it simple for windowing and applicable for strings as well - what about basing on string's length to keep it predictable. And keeping int-base for int values?

Reply:
I think it is fair to use int() conversion wherever possible and if it does not work do something else. Maybe default to time() as at least it will be more meaningful than length of a string for the example.

Reply:
Fixed that as per comment

Reply:
Thank you.